compose/mpp/demo/build.gradle.kts

```diff
@@ -162,7 +162,6 @@ kotlin {
                 implementation(project(":compose:material3:adaptive:adaptive-layout"))
                 implementation(project(":compose:material3:adaptive:adaptive-navigation"))
                 implementation(project(":compose:material:material"))
-                implementation(project(":compose:mpp"))
```

Comment:
I made this change to fix the build. It's weird since it works in jb-main as is.

Although this dependency is not needed. 

I'll think about it again later.

Reply:
fixed